### ISSUE TITLE
Error: [minor] DNG Version 1.6.0.0 not yet tested

### ISSUE THREAD

**Issue by vanja-bertalan** [exiftool/exiftool#135]
I'm running exiftool 12.41, changelog says that support for DNG version 1.6.0.0 was added in exiftool 12.33, yet I cannot get it to work.

This is what I've tried

```
$ exiftool -DefaultScale="4.0 3.0" BDJM0450.DNG
Error: [minor] DNG Version 1.6.0.0 not yet tested - BDJM0450.DNG
    0 image files updated
    1 files weren't updated due to errors

```

and

```
$ exiftool -ignoreMinorErrors -DefaultScale="4.0 3.0" BDJM0450.DNG
Warning: DNG Version 1.6.0.0 not yet tested - BDJM0450.DNG
    0 image files updated
    1 image files unchanged

```

<3

**Comment by boardhead** [comment]
Thanks.  I forgot to update this check when writing DNG 1.6 files.  I'll fix this in the next release.

- Phil

**Comment by boardhead** [comment]
(It is unfortunate that this test is necessary at all, but Adobe has done some stupid things in the past which have broken forward compatibility unnecessarily, so I have added this check just to be safe.)

**Comment by boardhead** [comment]
This was implemented on June 1, 2000 in version 12.42
